committee-steering/sig-governance.md

```diff
@@ -0,0 +1,533 @@
+lazy consensus models), all involved parties must be made aware of the
+proposal.  While it is the responsibility of involved parties to
+watch for new proposals and read them in a timely manner,
+it is the responsibility of the SIG governance to ensure that
+where and how are well defined.
+
+In order to avoid [Warnock's Dilemma][warnocks-dilemma], it is
+recommended at least one active member respond to a proposal before
+it is accepted.
+
+In SIG ~~Foo~~, proposals are made using a commonly used tool
+(issues, PRs, docs, etc) and then published to the SIG
+through kubernetes-sig~~foo~~-members@googlegroups.com mailing list with the
+subject containing either `KEP: ` or `RFC: `.  This provides a simple
+mechanism to subscribe to proposals that is less noisy than subscribing
+to github notifications.
+
+Discussion may occur on the issue / PR / doc itself.  Any comments
+on the doc are considered blocking until they are responded too.
+Once the comments are responded to, they are considered addressed
+unless either there is another response *or* the comment has some
+language to indicate it is a blocking issue that must be explicitly
+resolved.  e.g. *lets hold off on doing this until...*
+
+Once all comments have been addressed on the proposal without responses,
+another email
+must be sent to the mailing list with `LastCall: KEP: ` or
+`LastCall: RFC: `.  This is to ensure that everyone involved in
+the decision making process understands the proposal will be
+considered accepted if there are no further comments.  If
+there are no further comments or responses to the
+mail in 1-2 business days the proposal can be considered accepted.
+
+**Important:** Once a proposal has been accepted then the decision
+has been made and a commitment has been given to whoever implements the
+proposal that their contribution will be accepted if it is
+well implemented.  If it is deemed necessary to reverse a decision,
+this should be done as a new proposal and require lazy consensus on
+the reversal.
+
+In cases where consensus cannot be achieved in 1-2 weeks, the proposal
+may be escalated to the SIG ~~Foo~~ technical leads.  The leads should
+ensure that they are in agreement on the resolution.
+
+#### Defaulting toward action
+
+In cases where there is consensus that a problem exists, but a
+resolution cannot be achieved through consensus or escalation
+to technical leaders, then the default resolution should be
+that the individuals staffing the effort make the decision.  This
+is to avoid deadlock and stagnation when addressing known issues
+with multiple valid solutions.
+
+### Test health ownership and triaging issues
+
+#### Test dashboard
+
+SIG ~~Foo~~ maintains a dashboard on [testgrid](https://k8s-testgrid.appspot.com/sig-foo-master)
+displaying all of the tests the certify the SIG ~~Foo~~ area as
+healthy and the artifacts are in a releasable state.
+
+#### Oncall
+
+In order to facilitate consistently passing tests and triage of issues,
```

Comment:
This should be spelled out under the responsibilities of TL very clearly.

Reply:
I also think this should be a callout to the PL/FL/OL to route this... This seems more a priority for them then the TL.

Reply:
Do you mean the PL (final name TBD) role is to make sure the rotation is setup and has a healthy number of active folks and we should clarify this in the responsibilities section?  (I agree).  Or was there something else?